pandas/io/json/normalize.py

```diff
@@ -42,7 +44,15 @@ def nested_to_record(ds, prefix="", sep=".", level=0):
 
         .. versionadded:: 0.20.0
 
-    level: the number of levels in the jason string, optional, default: 0
+    level: the number of levels in the json string, optional, default: 0
```

Comment:
Can you run `scripts/validate_docstring.py` against this? Should give you some failures to correct